BMM.Core/ViewModels/SharedTrackCollectionViewModel.cs

```diff
@@ -39,36 +37,26 @@ public class SharedTrackCollectionViewModel : TrackCollectionViewModel, ISharedT
                 downloadQueue,
                 networkSettings)
         {
-            _userDialogs = userDialogs;
-            AddToMyPlaylistCommand = new ExceptionHandlingCommand(async () =>
-            {
-                try
-                {
-                    await Client.SharedPlaylist.Follow(_sharingSecret);
-                    _messenger.Publish(new PlaylistStateChangedMessage(this, MyCollection.Id));
-                    await CloseCommand.ExecuteAsync();
-                }
-                catch (FollowOwnTrackCollectionException)
```

Comment:
just checking if I get this correctly:
I don't have to wrap my code in a try-catch anymore. Instead I create the FollowOwnTrackCollectionExceptionHandler which is called whenever something in a guarded action throws a FollowOwnTrackCollectionException.
Correct?

Reply:
Oh. I actually define the exceptionhandlers in the action as well.